test/bin/mdx-test/expect/errors/test-case.md

```diff
@@ -30,38 +30,34 @@ Hi!
 
 
 ```ocaml version=4.02
-# let x =
-  1 + "42"
+# let x = 1 + "42";;
 Characters 14-18:
 Error: This expression has type bytes but an expression was expected of type
          int
 ```
 
 ```ocaml version=4.06
-# let x =
-  1 + "42"
+# let x = 1 + "42";;
 Characters 14-18:
 Error: This expression has type string but an expression was expected of type
          int
 ```
 
 ```ocaml version=4.07
-# let x =
-  1 + "42"
+# let x = 1 + "42";;
 Characters 14-18:
 Error: This expression has type string but an expression was expected of type
          int
 ```
 
 ```ocaml version>=4.08
-# let x =
-  1 + "42"
-Line 2, characters 7-11:
+# let x = 1 + "42";;
+Line 1, characters 13-17:
 Error: This expression has type string but an expression was expected of type
          int
 ```
 
-```ocaml non-deterministic=output
```

Comment:
No idea what is causing this to fail, I had this issue even without integrating the formatting of blocks, and this only involves the evaluation anyway...